recipe/meta.yaml

```diff
@@ -9,11 +9,10 @@ source:
   url: https://github.com/NVIDIA/nccl/archive/v{{ version }}.tar.gz
   sha256: 6b946b70a9d2d01871842cbd15ec56488d358abe9a0f3767e372fddc3e241ba7
   patches:
-    # Upstreaming w/PR: https://github.com/NVIDIA/nccl/pull/854
-    - 0001-Allow-custom-NVCC-path.patch
```

Comment:
Q: This patch was removed?

Reply:
yes, it's not needed, because [here](https://github.com/NVIDIA/nccl/blob/2ea4ee94bfb04c886c79ccae60ac9961000fdee2/makefiles/common.mk#L19) you can just set `CUDA_HOME` to `$BUILD_PREFIX` rather than setting `NVCC` directly

Reply:
The patch did this
```
-NVCC = $(CUDA_HOME)/bin/nvcc
+NVCC ?= $(CUDA_HOME)/bin/nvcc
```